pmgr/harvester.py

```diff
@@ -3,28 +3,29 @@
 # A bizarre combination of code from IocManager, the parent/child IOC compilation process, and
```

Comment:
I think this first line sums it up.

I'm not convinced this would work any more, and even if it did, our database is mostly populated at this point.

Let's kill this.

Reply:
Done in fc6f8b9